src/amo/containers/Home.js

```diff
@@ -34,6 +34,21 @@ export class HomePageBase extends React.Component {
     const { i18n } = this.props;
     return (
       <div className="HomePage">
+        <div className="HomePage-welcome">
+          <p className="HomePage-welcome-text">
+            {i18n.gettext(dedent`Extensions are special features you can add to Firefox.
+            Themes let you change your browser's appearance.`)}
+          </p>
+          <div className="HomePage-welcome-links">
+            <Link className="HomePage-extensions-link" to="/extensions/">
```

Comment:
This is pedantic, but I've been using `visibleAddonType(ADDON_TYPE_EXTENSION)` instead of hardcoding `extensions`, just to avoid magic constants. I guess URLs shouldn't change but it seems a bit nicer. 🤷‍♀️